script.js

```diff
@@ -0,0 +1,151 @@
+// import { cors } from 'cors';
+
+const express = require('express');
+const cors = require('cors');
+const app = express();
+const ulid = require('ulid');
+
+const port = 3000;
+const redis = require('redis');
```

Comment:
I always recommend starting a file by placing all import statements to increase readability